harness/determined/util.py

```diff
@@ -20,6 +20,11 @@
 
 @util.preserve_random_state
 def download_gcs_blob_with_backoff(blob: Any, n_retries: int = 32, max_backoff: int = 32) -> Any:
+    from google.cloud.storage import Blob
```

Comment:
We try to stick to the [Google Python Style Guide for imports](https://google.github.io/styleguide/pyguide.html#22-imports) in all new code.  Basically: import the module for a symbol rather than the symbol itself.

So this should be:
```python
from google.cloud import storage
...
assert isinstance(blob, storage.Blob), ...
```